modules/grafana/local/env.list

```diff
@@ -0,0 +1,57 @@
+# set of environment variables passed into docker for initializing grafana datasources
```

Comment:
Is there a better way todo this? For example, can we have common read-only creds for clusters instead?

I tend not to like things like this because:
- It exposes mintel-specific information in a public repo
- If we add/remove clusters, we then have to "remember" to update this list

Reply:
Moved this to https://gitlab.com/mintel/satoshi/tools/build-harness-extentions-private. I've kept this file the same there for now. I agree that this is not a nice implementation where the names are all listed in one file, but I think that we will be adding new data-sources rarely and I want to get something out that works by this friday if possible. If this is a big issue can take the time to refactor further (could do something like get the names from the AWS secretsmanager keys, etc.), but these names need to somehow be listed in a file to pass to docker with the --env-file flag in the current implementation